nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
```

Comment:
Should there be a more explicit link in this article to this sample than just a related link?